src/components/Permissions/PermissionsTab.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react'
+import Page from 'components/Page'
+import { useI18n } from 'cozy-ui/transpiled/react'
+import { Tabs, Tab } from 'cozy-ui/transpiled/react/MuiTabs'
+import AppList from './AppList'
+import DataList from './DataList'
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import { routes } from 'constants/routes'
+
+function a11yProps(index) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`
+  }
+}
+
+const PermissionsTab = () => {
+  const { t } = useI18n()
+  const [tab, setTab] = useState('apps')
+  const handleChange = (event, value) => setTab(value)
+
+  return (
+    <Page narrow>
+      <Tabs value={tab} onChange={handleChange}>
+        <Tab
+          value="apps"
+          label={t('Permissions.applications')}
+          href={`#${routes.appList}`}
+          {...a11yProps(0)}
+        />
+        <Tab
+          value="data"
+          label={t('Permissions.data')}
+          href={`#${routes.dataList}`}
+          {...a11yProps(1)}
+        />
+      </Tabs>
+      <Divider className="u-mb-1" />
+      {tab === 'apps' ? <AppList /> : null}
+      {tab === 'data' ? <DataList /> : null}
```

Comment:
Can't you use 
```
<TabPanel index="apps"><AppList /><TabPanel>
<TabPanel index="data"><DataList /><TabPanel>
```
or something like that?

Reply:
I was inspired by this code which seems more recent: [MuiTabs ](https://github.com/cozy/cozy-ui/tree/67485ca3a6db28dd4ba64d2c91188c21b00034ce/react/MuiTabs )(2021) || [TabPanel](https://github.com/cozy/cozy-ui/blob/67485ca3a6db28dd4ba64d2c91188c21b00034ce/react/README.md) (2020)

Reply:
I see the misunderstanding: I thought that mui was exporting a TabPanel component since they use it on their exemple: https://v4.mui.com/components/tabs/#tabs  : 
```jsx
<AppBar position="static">
  <Tabs value={value} onChange={handleChange} aria-label="simple tabs example">
    <Tab label="Item One" {...a11yProps(0)} />
    <Tab label="Item Two" {...a11yProps(1)} />
    <Tab label="Item Three" {...a11yProps(2)} />
  </Tabs>
</AppBar>
<TabPanel value={value} index={0}>
  Item One
</TabPanel>
<TabPanel value={value} index={1}>
  Item Two
</TabPanel>
<TabPanel value={value} index={2}>
  Item Three
</TabPanel>
```

But in fact when checking the full source, TabPanel is a custom component: 
```jsx
return (
    <div
      role="tabpanel"
      hidden={value !== index}
      id={`simple-tabpanel-${index}`}
      aria-labelledby={`simple-tab-${index}`}
      {...other}
    >
      {value === index && (
        <Box p={3}>
          <Typography>{children}</Typography>
        </Box>
      )}
    </div>
  );
```

I think we should use something like this component at least for A11y reason

Reply:
I don't understand which component I should use

Reply:
```jsx
const TabPanel = props => {
const { children, value, index, ...other } = props;
return  ( 
    <div role="tabpanel" 
    hidden={value !== index} 
    id={`simple-tabpanel-${index}`} 
    aria-labelledby={`simple-tab-${index}`} {...other} > 
     {value === index && ( {children} )} 
    </div> );
}
```

```jsx
<Tabs value={tab} onChange={handleChange}>
        <Tab
          value="apps"
          label={t('Permissions.applications')}
          href={`#${routes.appList}`}
          {...a11yProps(0)}
        />
        <Tab
          value="data"
          label={t('Permissions.data')}
          href={`#${routes.dataList}`}
          {...a11yProps(1)}
        />
      </Tabs>
      <Divider className="u-mb-1" />
<TabPanel value={tab} index="apps">
 <AppList />
<TabPanel> 
<TabPanel value={tab} index="data">
    <DataList />
<TabPanel>
``

Reply:
Thank you